sources/packages/backend/apps/db-migrations/src/sql/Restrictions/Rollback-update-restrictions-review-impact.sql

```diff
@@ -0,0 +1,76 @@
+-- Revert the updates for the restriction code 5, 7 and 9.
+UPDATE
+    sims.restrictions
+SET
+    action_type = ARRAY ['Stop full time disbursement'] :: sims.restriction_action_types []
+WHERE
+    restriction_code IN ('5', '7', '9');
+
+-- Revert the updates for the restriction code 12, O and PTWTHD.
+UPDATE
+    sims.restrictions
+SET
+    action_type = ARRAY ['Stop part time disbursement', 'Stop full time disbursement'] :: sims.restriction_action_types []
+WHERE
+    restriction_code IN ('12', 'O', 'PTWTHD');
+
+-- Revert the updates for the restriction code APPC.
+UPDATE
+    sims.restrictions
+SET
+    restriction_code = 'APPC',
+    action_type = ARRAY ['Stop full time disbursement', 'Stop part time disbursement'] :: sims.restriction_action_types []
+WHERE
+    restriction_code = 'HOLD';
+
+-- Revert the updates for the restriction code B6
+UPDATE
+    sims.restrictions
+SET
+    restriction_type = 'Federal',
+    description = 'Bankruptcy restriction on file prevent CSL/BCSL disbursements.',
+    restriction_category = 'Federal'
+WHERE
+    restriction_code = 'B6';
+
+-- Revert the updates for the restriction code B6A
+UPDATE
+    sims.restrictions
+SET
+    description = 'For reinstatment of your BCSL eligibility please complete an appeal.',
```

Comment:
Please fix the typo "reinstatement". No business confirmation is needed.

Reply:
Updated.